src/components/patch/Patch.less

```diff
@@ -7,8 +7,13 @@
   padding-right: 100px;
 }
 
-.patch-search-bar {
-  text-align: center;
-  padding-top: 50px;
-  padding-bottom: 50px;
+.search-container {
```

Comment:
Was the reason for changing the alignment approach so that you could left-justify the label text? If so I think a better approach would be to keep text-align:center on the container, but override it for the input element. That way you have a more consistent styling hierarchy for all screen sizes. It also lets you set a max-width for the input (which I think we should do since it's huge on my 26 inch monitor)

Reply:
I set a max width, which definitely does look better, but I'm having trouble figuring out how to center align the search bar now.

Reply:
Leaving this alone for now since we discovered the CSS is mysterious